index.html

```diff
@@ -2959,6 +2959,14 @@ <h3><code>forms</code></h3>
 In the case of operation type <code>writeallproperties</code>, it is expected that the <a>Consumer</a> provides ALL writable (non <code>readOnly</code>) properties and the (new) assigned values (e.g., within payload). Otherwise the <a>Thing</a> MAY refuse this call to avoid inconsistency.
 </span></p>
 
+<p><span class="rfc2119-assertion" id="td-writemultiple-consumer">
```

Comment:
we are not allowerd to create a new assertion. We should change this to a 'information' for developer.